test/controllers/invitations/InviteControllerSpec.scala

```diff
@@ -146,10 +145,10 @@ object InviteControllerSpec extends SpecBase with JsonFileReader with MockitoSug
     override def getSchemeDetailsRefresh(psaId: String,
                                          idNumber: String,
                                          schemeIdType: String)
-                                    (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] = ???
+                                        (implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Unit] = ???
```

Comment:
There's a missing implementation here I think

Reply:
There are a couple of these in this file. These override/implements a base member which are present in some controllers or spec files so I haven't changed any of these.

Reply:
they created the minimum implementation of a connector to test it's some functionality. It was not in scope.